As per README.md, to run edge UT & IT, execute "make test" & "make integration_test", these commands fail

**What happened**:
As per README.md, after executing "make test" & "make integration_test", the execution fails with error.

ubuntu@ubuntu1804VM:~/gopath/src/github.com/kubeedge/kubeedge/edge$ cd ..
ubuntu@ubuntu1804VM:~/gopath/src/github.com/kubeedge/kubeedge$ make test
make: *** No rule to make target 'test'.  Stop.

ubuntu@ubuntu1804VM:~/gopath/src/github.com/kubeedge/kubeedge$ make integration_test
make: *** No rule to make target 'integration_test'.  Stop.

**What you expected to happen**:
After execution of the command, UT & IT should had successfully run.

**How to reproduce it (as minimally and precisely as possible)**:
Simply go to ~/gopath/src/github.com/kubeedge/kubeedge path and execute "make test"
Simply go to ~/gopath/src/github.com/kubeedge/kubeedge path and execute "make integration_test"

**Anything else we need to know?**:

**Environment**:
- KubeEdge version: 0.1
- Hardware configuration: NA
- OS (e.g. from /etc/os-release): "Ubuntu 18.04.1 LTS"
- Kernel (e.g. `uname -a`): NA
- Others:

--- a/README.md
+++ b/README.md
@@ -153,7 +153,7 @@ If you are using HuaweiCloud IEF, then the edge node you created should be runni
 ### Run Edge Unit Tests
 
  ```shell
- make test
+ make edge_test
  ```
  To run unit tests of a package individually 
  ```shell
@@ -165,8 +165,7 @@ If you are using HuaweiCloud IEF, then the edge node you created should be runni
 ### Run Edge Integration Tests
 
 ```shell 
-cd $GOPATH/src/github.com/kubeedge/kubeedge/edge
-make integration_test
+make edge_integration_test
 ```
 
 ### Details and use cases of integration test framework

